examples/kebechet/alerting.md

```diff
@@ -0,0 +1,29 @@
+# Alerting
+
+Once we have all metrics collected and stored, we can define the alerts that are going to be triggered. These alerts can be provided by:
+
+- Github Issues, using Prometheus GitHub AlertManager.
+
+- email, using Prometheus webhooks.
```

Comment:
similarly add a description specifying how the alert can be routed to an email and maybe include screenshots?

Reply:
I think we can remove this actually, as this was used only internally.